graftm/hmmer.py

```diff
@@ -545,144 +490,212 @@ def extract_orfs(self, input_path, orfm, orf_out_path):
         logging.debug("Running command: %s" % cmd)
         subprocess.check_call(cmd, shell=True)
     
-    @timeit
-    def p_search(self, files, args, base, unpack, raw_reads, search_method, gpkg):
+    def _get_read_names(self, search_result, max_range):
         '''
-        Protein search pipeline - The searching step for the protein 
-        pipeline, where hits are identified in the input reads through 
-        searches with hmmer or diamond.
+        _get_read_names - loops through hmm hits and their alingment spans to
```

Comment:
I think there is no need to repeat the name of the method - it is right there already.